src/components/lab/NumberInput/NumericArrows/NumericArrows.scss

```diff
@@ -0,0 +1,49 @@
+@use '../../../../../styles/mixins';
+@use '../../../variables';
+@use '../../../controls/mixins.scss' as control-mixins;
+@use '../../../controls/variables.scss' as control-variables;
+
+$block: '.#{variables.$ns}numeric-arrows';
+
+#{$block} {
+    --_--numeric-arrows-border-color: var(
+        --_-g-number-input-numeric-arrows-border-color,
+        var(--g-text-input-border-color, transparent)
+    );
+
+    --_--border-width: var(--g-text-input-border-width, 1px);
+    border-style: solid;
```

Comment:
Seems like these styles are duplicated

Reply:
Which styles and where is their duplicate is located?

Reply:
border styles between NumericArrows.scss and NumberInput.scss